README.md

```diff
@@ -63,6 +64,16 @@ Key generators are the strategies used to generate the unique revision key. Curr
 
 This key generator will fingerprint the `index.html` and return an MD5 hash. The generation of the revision key, using this generator, is guaranteed to be idempotent. That is, the same revision key will be generated for the same set of project files. If the project files change in any way, this will be reflected by a change in the revision key.
 
+### Git Commit generator
```

Comment:
@alisdair Based on the naming of the Version Commit generator (which creates a key based on the `version` and the `commit` hash), should this one be something along the lines of Tag Commit generator?